### ISSUE TITLE
Post Login  - Accept

### ISSUE THREAD

**Issue by collinz1** [danwalkeruk/fortigate2csv#1]
Dan
This script is great, i have customised to suit my needs but this is going to save hours in the future.  The script assumes that the login process to the FG has no login banners, in my case we are using post login banners that must be accepted before the login is verified.  Is there any way to add an option / variable that will simulate the pressing of the Accept button once credentials have been submitted?
Thanks

**Comment by danwalkeruk** [comment]
Hi @collinz1 

Thanks for the feedback, I wasn't actually aware of that functionality within the FortiGate UI!

Would you be able to provide a copy of the HTTP response from FortiGate after the POST is sent to `https://{host}/logincheck`? 

Also, is the `ccsrftoken` returned in that initial banner response? Or does that come after the accept button is clicked?

**Comment by collinz1** [comment]
Dan
i have attached the HTTP response and the HTML that is returned (not sure how useful that is though).
Looks like the ccsrftoken is after the accept button is clicked but i could be wrong.
FYI - post login banner is enabled using the following:

_config system global
     set post-login-banner enable
end_

[http_response.txt](https://github.com/danwalkeruk/fortigate2csv/files/5578740/http_response.txt)
[html_source.txt](https://github.com/danwalkeruk/fortigate2csv/files/5578744/html_source.txt)

**Comment by danwalkeruk** [comment]
Hi @collinz1 

I replicated the login banner issue with a test device. The CSRF token is received with the login banner, but access isn't permitted until a POST is made to `/logindisclaimer` with the appropriate values.

I've pushed a fix to this which I've tested and can confirm works. If a login banner is presented, it will be accepted with a message in the terminal to show that.

Let me know if you experience an issue with the new code.
